### PR TITLE
Opencv stuff

### PR DESCRIPTION
**Pull request by rmeertens**
Now it is easy to convert back to the normally used yuv422 arrays of paparazzi. 

Any place better by the way for these conversion functions?

### CODE REVIEW

**Comment by fvantienen** [comment]
the `lib/image.h` and `lib/image.c`?

**Comment by flixr** [comment]
I would put it in separate files, you don't want to have the OpenCV dependency if you "just" need the paparazzi specific stuff in image.h

**Comment by rmeertens** [comment]
Ok, will make seperate files out of them.

**Comment by rmeertens** [comment]
Fixed!

**Comment by flixr** [comment]
Can you plz also add the license and doxygen headers to the new files?

File: sw/airborne/modules/computer_vision/opencv_image_functions.h [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+#ifndef OPENCV_IMAGE_FUNCTIONS_H
+#define OPENCV_IMAGE_FUNCTIONS_H
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+using namespace cv;
```
Comment (flixr) [inline sw/airborne/modules/computer_vision/opencv_image_functions.h]: Also it's good style to not use namespaces in the header files, instead use `cv::Mat` below.

**Comment by rmeertens** [comment]
Done.

**Pull request merged**